coqpp/coqpp_parse.mly

```diff
@@ -391,6 +391,9 @@ rules:
 rule:
 | symbols_opt ARROW CODE
   { { gprod_symbs = $1; gprod_body = $3; } }
+| symbols
+  { { gprod_symbs = $1;
+      gprod_body = { code = "()"; loc = { loc_start = Lexing.dummy_pos; loc_end = Lexing.dummy_pos } }; } }
```

Comment:
This commit should probably be removed from this PR considering the discussion in https://github.com/coq/coq/pull/14071#discussion_r642929069

Reply:
Done.